core/runtime.rs

```diff
@@ -1770,6 +1788,14 @@ impl JsRuntime {
       .map(|handle| v8::Local::new(tc_scope, handle))
       .expect("ModuleInfo not found");
     let mut status = module.get_status();
+
+    // FIXME(bartlomieju): this is a hack, when snapshotting we shouldn't
+    // be trying to evaluate same module multiple times
+    if status == v8::ModuleStatus::Evaluated {
+      let (sender, receiver) = oneshot::channel();
+      sender.send(Ok(())).unwrap();
+      return receiver;
+    }
```

Comment:
This should be fixed in a separate PR, https://github.com/denoland/deno/issues/17764 should resolve it